pkg/vm/vm.go

```diff
@@ -991,6 +1013,15 @@ func (v *VM) execute(ctx *Context, op Instruction) {
 	}
 }
 
+func cloneItem(item StackItem) StackItem {
```

Comment:
`cloneIfStruct`? And it would be nice to simplify `APPEND` implementation by using it.